src/Illuminate/Support/helpers.php

```diff
@@ -129,6 +129,19 @@ function array_flatten($array, $depth = INF)
     }
 }
 
+if(! function_exists("array_flatten_keeping_keys")) {
+    /**
+     * @param $array
+     * @param string $prefix
+     * @return array
+     * Thanks
+     * https://stackoverflow.com/questions/9546181/flatten-multidimensional-array-concatenating-keys/9546215#9546215
+     */
+    function array_flatten_keeping_keys($array, $prefix = '') {
```

Comment:
You already defined a helper with the same name.
And BTW helpers are added as needed, I do not think we add them right away.

Reply:
kk
next time I know! Thanks